### PR TITLE
Norwegian Bokmål language updated

### PR DESCRIPTION
**Pull request by comradekingu**
An update on #7, but really from scratch.

As always it would be great to have Mauth in
https://hosted.weblate.org/trial/

Add "kingu" if you want help setting it up.

I can put in a PR to fix the source strings afterwards :)

### CODE REVIEW

File: app/src/main/res/values-nb-rNO/strings.xml [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+    <string name="home_sort_issuer_descending">Utsteder (Å–A)</string>
+    <string name="home_sort_date_ascending">Eldste dato først</string>
+    <string name="home_sort_date_descending">Nyeste dato først</string>
 
     <string name="qrscan_title">Skann en QR-kode</string>
     <string name="qrscan_permissions_title">Manglende tilganger</string>
-    <string name="qrscan_permissions_subtitle">Kameratilgang er ikke innvilget.</string>
-    <string name="qrscan_permissions_subtitle_rationale">Kameratilgang kreves for at programmet skal kunne analysere QR-koden. Uten det kan ikke Mauth skanne og importere kontoene dine.</string>
+    <string name="qrscan_permissions_subtitle">Kameratilgang ikke innvilget.</string>
+    <string name="qrscan_permissions_subtitle_rationale">Kameratilgang kreves for å skanne og analysere QR-kodeR.</string>
```
Comment (comradekingu) [inline app/src/main/res/values-nb-rNO/strings.xml:33]:
```suggestion
    <string name="qrscan_permissions_subtitle_rationale">Kameratilgang kreves for å skanne og analysere QR-koder.</string>
```

**Comment by X1nto** [comment]
Do you mind if I postpone merging until I merge the account export branch? It's almost complete. https://github.com/X1nto/Mauth/tree/feature/account-export

**Comment by comradekingu** [comment]
Happy if you merge it at all :)
Ping me if you see me not fixing conflicts.

**Comment by X1nto** [comment]
Hi! I've updated the master branch with new strings, you can safely update the translation, as nothing should change until the next release

**Comment by comradekingu** [comment]
@X1nto There we go.
Probably better ways to do what I just did, but squash and merge removes all crime.

**Pull request merged**